tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -53,6 +55,40 @@ pub(crate) struct TestCommand {
     /// Number of threads used for running tests in parallel
     #[clap(long, default_value_t = rayon::current_num_threads())]
     test_threads: usize,
+
+    /// Configure formatting of output
+    #[clap(long)]
+    format: Option<Format>,
+
+    /// Display one character per test instead of one line
+    #[clap(short = 'q', long = "quiet", action)]
+    quiet: bool,
```

Comment:
These two flags can contradict each other and should be made mutually exclusive.

Reply:
Good point. I just tried it in Rust and it seems `format` takes precedence over `-q`. Should we do the same? We could also drop this flag and just keep `format`.

Reply:
Ah yes, like [these](https://github.com/noir-lang/noir/blob/303a9c1c88256bc6714490e42a5641ac903eeb89/tooling/nargo_cli/src/cli/mod.rs#L64)

What does the `action` do?

Reply:
FWIW I like `-q`

Reply:
Oh, it's a left-over. I first wanted it to be `Option<bool>` but then you have to pass `--quiet true` or `--quiet false` and I saw `action` could solve that, but I ended up not using it.